web/templates/settings/account_settings.hbs

```diff
@@ -77,9 +77,12 @@
                       }}
                 </div>
                 {{#if owner_is_only_user_in_organization}}
-                    <button type="submit" class="button rounded button-danger deactivate_realm_button">
-                        {{t 'Deactivate organization' }}
-                    </button>
+                    {{> ../components/action_button
+                      label=(t "Deactivate organization")
+                      type="quiet"
+                      intent="danger"
+                      custom_classes="deactivate_realm_button inline-block"
```

Comment:
Why does this one have an `inline-block` class?

Reply:
I guess it's to arrange this:

![image](https://github.com/user-attachments/assets/eb9f5523-dec4-4f5c-9f6b-9979b89e57fe)

Seems OK.